liip/imagine-bundle/1.7/etc/routing/imagine.yaml

```diff
@@ -0,0 +1,19 @@
+liip_imagine_filter: 
```

Comment:
Assuming adding the following addition to the `manifest.json` results in the same auto-registration of the file, I agree, aside from my comment below:

```yaml
"copy-from-package": {
    "Resources/config/routing.xml": "%ETC_DIR%/routing/"
}
```

My only concern is that the use of YAML here was an intentional choice made to lower the entry barrier for someone looking to, perhaps, edit the route paths; XML is generally more difficult for newcomers to follow or edit.